changelog/5574.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed issue where metadata was not being sent with the tracker from session_start event when calling action server.
```

Comment:
Fixed an issue that happened when metadata is passed in a new session.

Now the metadata is correctly passed to the ``ActionSessionStart``.